modules/graceful-shutdown.adoc

```diff
@@ -19,6 +24,19 @@ It is important to take an etcd backup before performing this procedure so that
 
 .Procedure
 
+. If you are shutting the cluster down for an extended period, note the date on which certificates expire.
++
+[source,terminal]
+----
+$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
+----
++
+.Example output
+----
+2022-08-05T14:37:50Zuser@user:~ $ <1>
+----
+<1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates.
```

Comment:
I can't remember if we talked about this, but the recovering from expired certs section [1] was recently updated to add additional CSRs that might need to approved (for UPI installs). So it might not only be `node-bootstrapper` CSRs. Suggest leaving the specifics out and just saying "... to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates."

[1] https://docs.openshift.com/container-platform/4.8/backup_and_restore/disaster_recovery/scenario-3-expired-certs.html#dr-scenario-3-recovering-expired-certs_dr-recovering-expired-certs